tools/launch_release/build/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM python:3.9-slim-bullseye
+LABEL maintainer='Weights & Biases <support@wandb.com>'
+
+# Install Go.
+COPY --from=golang:1.23-alpine /usr/local/go/ /usr/local/go/
+ENV PATH="/usr/local/go/bin:${PATH}"
+
+# Set Go environment paths
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+# Set up Go workspace directories
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+
+# Install Rust using Rustup
+RUN apt-get update && apt-get install -y curl && \
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
+    apt-get remove -y curl && apt-get -qy autoremove && \
+    apt-get clean && rm -r /var/lib/apt/lists/*
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+
+# install git
+RUN apt-get update && apt-get upgrade -y \
+    && apt-get install -y git gcc python3-dev \
+    && apt-get -qy autoremove \
+    && apt-get clean && rm -r /var/lib/apt/lists/*
+
+# required pip packages
+ARG REF="main"
+RUN pip install --upgrade pip && \
+    pip install --upgrade "setuptools>=70.0.0" && \
```

Comment:
we don't use setuptools anymore, we use `hatchling` is it used for something else?

Reply:
oh interesting, I added this to resolve an issue vulnerability issue contained in  0.58.1 which is shipped with the base container. Will test removing instead